Remotion/Core/Tools/AppDomainAssemblyResolver.cs

```diff
@@ -30,14 +30,14 @@ namespace Remotion.Tools
   [Serializable]
   public sealed class AppDomainAssemblyResolver : MarshalByRefObject
   {
-    public static AppDomainAssemblyResolver CreateInAppDomain (AppDomain appDomain, string applicationBase)
+    public static AppDomainAssemblyResolver? CreateInAppDomain (AppDomain appDomain, string applicationBase)
```

Comment:
I don't think we expect/allow null here.

Reply:
`AppDomain.CreateInstanceFromAndUnwrap(...)` is nullable (https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/AppDomain.cs#L382).

Reply:
Yes, but our code doesn't do null here. todo-ticket for a null-guard and not-null in the signature, please.